### ISSUE TITLE
V1.1.9 startup problem with Node 6.7.0

### ISSUE THREAD

**Issue by tengelmeier** [lprhodes/homebridge-broadlink-rm#48]
I can't start after an npm update -g :

```
 ERROR LOADING PLUGIN homebridge-broadlink-rm:
[4/16/2017, 6:07:12 PM] /usr/local/lib/node_modules/homebridge-broadlink-rm/accessories/accessory.js:34
  async setCharacteristicValue (props, on, callback) {
        ^^^^^^^^^^^^^^^^^^^^^^
SyntaxError: Unexpected identifier
    at Object.exports.runInThisContext (vm.js:76:16)
    at Module._compile (module.js:528:28)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/homebridge-broadlink-rm/accessories/aircon.js:1:92)
    at Module._compile (module.js:556:32)

```

Strange enough: 
- I can start and load the plugin on a local Mac installation (learn_ir works, the buttons appear, however sent IR codes don't work on the target device)

- loading the plugin with the same configuration fails on a Raspberry Pi. 
         
     npm list -g 

shows identical versions at least for homebridge and  homebridge-broadlink-rm:

Raspberry:

```
├─┬ homebridge@0.4.17
....
├─┬ homebridge-broadlink-rm@1.1.9
│ └─┬ broadlinkjs@0.1.0 (git+https://github.com/lprhodes/broadlinkjs.git#a463147a6a71855722221da0f74adcba86d7bb5b)
│   └── @types/node@7.0.12
```

Mac:  (with a local 1.1.9 homebridge-broadlink-rm installation):

```
├─┬ broadlinkjs@0.1.0 (git+https://github.com/lprhodes/broadlinkjs.git#a463147a6a71855722221da0f74adcba86d7bb5b)
│ └── @types/node@7.0.12
├─┬ homebridge@0.4.17
```

**Comment by tengelmeier** [comment]
Solved: The Raspberry used node 6.7.0, the Mac 7.6.0. 
It seems the module requires a more recent NodeJS version.

**Comment by lprhodes** [comment]
It certainly does - it's worth checking release notes for updates (https://github.com/lprhodes/homebridge-broadlink-rm/releases/tag/1.1.0). I'm not looking to add support for earlier versions, sorry.
